packages/core/security/core-security-server-internal/src/fips/fips.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SecurityServiceConfigType } from '../utils';
+
+export function isFipsEnabled(config: SecurityServiceConfigType): boolean {
+  return config?.experimental?.fipsMode?.enabled || false;
```

Comment:
NIT: `?? false` is technically more accurate

Reply:
Fixed in https://github.com/elastic/kibana/pull/181187/commits/7b7ed010889b5072d430622be372d4af9bfee487